README.md

```
@@ -0,0 +1,58 @@
+```bash
+npm run build
+```
+
+## 🛠️ Technology Stack
+- **Language**: TypeScript
+- **Build Tool**: Vite
+- **Game Engine**: TBD (Babylon.js or Phaser)
+- **Platform**: Web (with potential for desktop/console ports)
+
+## 📁 Project Structure
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)